### PR TITLE
Add accessible description to more dialogs

### PR DESCRIPTION
**Pull request by CoBC**

### CODE REVIEW

**Comment by amirmahdifard** [comment]
@CoBC i think for server propertis dialogue, it should be propertis for %1 rather than propertis of %1: so propertis for little friends tt for example

**Comment by amirmahdifard** [comment]
@CoBC what about server statistix wich is shift f9, and view channel information dialogue wich is f3?

**Comment by CoBC** [comment]
I forgot server statistics (now done), I already worked on channel info

File: Client/qtTeamTalk/channeldlg.cpp [hunk 1]
```diff
@@ -171,6 +173,7 @@ ChannelDlg::ChannelDlg(ChannelDlgType type, const Channel& chan, QWidget * paren
     break;
     case CHANNEL_READONLY :
         setWindowTitle(tr("View Channel Information"));
+        this->setAccessibleDescription(tr("View %1 information").arg(_Q(m_channel.szName)));
```
Comment (bear101) [inline Client/qtTeamTalk/channeldlg.cpp:176]: Remember that root channel will not have a name (szName == "").

**Comment by amirmahdifard** [comment]
@CoBC nice, but you only said server statisticks: please also mention the server name: for example, server statisticks for %1 or server statisticks of %1 that way i like it more bettar.

**Comment by amirmahdifard** [comment]
@CoBC also, please add an accessible discription to connect to a server dialogue: in classic client, if you go to connect to a server dialogue, it has accessible discription, it will say host manager: but in Qt client, since it has no accessible discription, as soon as you go to the connect to a server dialogue, your screen reader will start reading some random information at the same time as it does will all dialogues befor you add accessible discription to them.

**Comment by amirmahdifard** [comment]
@CoBC really thanks but please mention server name for server statistics accessible discription: please

**Comment by amirmahdifard** [comment]
@CoBC it's more than 24 hours after i commented and you even saw it: wy you don't want to add server name for server statistics? like, statistics for %1 or statistics of %1 ? thanks.

**Comment by CoBC** [comment]
For now we don't retrieve server properties in server stats dialog, and 
I find this a big chane just to add server name in accessible description.

**Comment by amirmahdifard** [comment]
@CoBC ok thanks for the answer: i just suggested you to do that because you did it for server propertis, but ok, fare enough.

**Comment by amirmahdifard** [comment]
@CoBC can you please explane wich one is vidio text dialogue that you added discription for? i don't understand wich one it is: thanks

**Comment by CoBC** [comment]
To see this dialog go to preferences/display and here you will find a button to configure video window, it is this dialog

**Comment by amirmahdifard** [comment]
@CoBC wow! nice that you added for that dialogue, i didn't realised that lol because you just said vidio text dialogue: but thanks for the answer, good job,  and very nice!

**Pull request merged**